### PR TITLE
#376 - Allow more custom HTML character codes

### PR DESCRIPTION
**Pull request by dexster**
Allows more HTML character codes to be added to custom codes as per #376

### CODE REVIEW

**Comment by dexster** [comment]
@probablyup Will this be merged or is there something that needs to be actioned?

**Comment by probablyup** [comment]
@dexster could you drop your changes to the docs files?

**Comment by dexster** [comment]
Doc updates removed

File: docs/markdown-to-jsx.js [hunk 1]
```diff
@@ -1,2 +0,0 @@
...l,"It's lightweight, customizable, and happily supports React-like libraries.")),a.default.createElement(Ln,null,"See the"," ",a.default.createElement("a",{target:"_blank",href:"https://github.com/probablyup/markdown-to-jsx/blob/master/README.md",rel:"noopener noreferrer"},"project README")," ","for detailed installation & usage instructions.")),a.default.createElement(Fn,null,a.default.createElement(Tn,{onInput:r,value:n}),a.default.createElement(Pn,null,a.default.createElement(kn,{options:Xn},n))))}(dn=sn||(sn={}))[dn.MAX=0]="MAX",dn[dn.HIGH=1]="HIGH",dn[dn.MED=2]="MED",dn[dn.LOW=3]="LOW",dn[dn.MIN=4]="MIN";var $n="rgba(255, 255, 255, 0.5)",zn="#fefefe",Cn=t.createGlobalStyle(jn(),$n,zn,$n,G($n,.75),G($n,.05)),_n=i.default.header(Hn()),Gn=i.default.p(Sn()),Ln=i.default.p(On(),T(.2,zn)),Rn=t.css(Mn()),Fn=i.default.section(En()),Tn=i.default.textarea(In(),Rn,G($n,.05)),Pn=i.default.div(xn(),Rn),Dn=i.default.button(wn()),Xn={overrides:{MyComponent:{component:function(e){return a.default.createElement(Dn,l({},e,{onClick:function(){alert("Look ma, I'm a real component!")}}))}}}};o.default.render(a.default.createElement(An,null),document.getElementById("root"))}(React,ReactDOM,styled);
```
Comment (probablyup) [inline docs/markdown-to-jsx.js:1]: Can you drop these file deletions from your PR please

**Comment by probablyup** [comment]
I'm interested in landing this if someone wants to update / recreate this PR against main

**Comment by probablyup** [comment]
Superceded by #502, thank you for contributing!

**Pull request closed without merging**